### ISSUE TITLE
create-react-app and Wallaby gives "setGlobal is not a function"

### ISSUE THREAD

**Issue by pauleveritt** [wallabyjs/public#976]
### Issue description or question

I'm using the latest create-react-app and followed the directions under the 
[create-react-app section](https://wallabyjs.com/docs/integration/react-jsx.html). I got the following exception when running my configuration:

```
TypeError: setGlobal is not a function
    at runTest (/Users/pauleveritt/projects/scratchpad/hello-create-react/node_modules/jest-cli/build/runTest.js:50:3)
    at promise.then (/Users/pauleveritt/projects/scratchpad/hello-create-react/node_modules/jest-cli/build/TestRunner.js:304:14)
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

I went to jest-cli's `runTest.js` line 50 and simply commented out that line. When you follow it to its source, it looks like something that simply inserts a copyright comment.

On one hand, it's hard to imagine how it is Wallaby's fault. OTOH, the regular create-react-app test runner's invocation of jest-cli doesn't seem affected.

### Wallaby.js configuration file

```javascript
module.exports = function (wallaby) {

    // Babel, jest-cli and some other modules are located under
    // react-scripts/node_modules, so need to let node.js know about it
    var path = require('path');
    process.env.NODE_PATH +=
        path.delimiter +
        path.join(__dirname, 'node_modules') +
        path.delimiter +
        path.join(__dirname, 'node_modules/react-scripts/node_modules');
    require('module').Module._initPaths();

    // Babel needs this
    process.env.NODE_ENV = 'development';

    return {
        files: [
            'src/**/*.js',
            '!src/**/*.test.js'
        ],

        tests: ['src/**/*.test.js'],

        env: {
            type: 'node',
            runner: 'node'
        },

        compilers: {
            '**/*.js': wallaby.compilers.babel({
                babel: require('babel-core'),
                presets: ['react-app']
            })
        },

        setup: (wallaby) => {
            wallaby.testFramework.configure({
                // as in node_modules/react-scripts/utils/createJestConfig.js
                moduleNameMapper: {
                    '^.+\\.(jpg|jpeg|png|gif|svg)$': require.resolve('react-scripts/config/jest/fileTransform.js'),
                    '^.+\\.css$': require.resolve('react-scripts/config/jest/cssTransform.js')
                }
            });
        },

        testFramework: 'jest'
    };
};
```

### Code editor or IDE name and version

WebStorm 2017.1 EAP

### OS name and version

OSX

**Comment by ArtemGovorov** [comment]
Just tried form the scratch with the latest `create-react-app` and can't reproduce the issue. The latest `create-react-app` right now is shipped with/installs `jest-cli@17.0.2` and this version of `jest` doesn't any `setGlobal` calls in the mentioned file.

Perhaps you've installed the latest `jest-cli` (v18.x) on top of the fresh `create-react-app` bootstrapped app? Try re-installing it from the scratch to see if it fixes the issue. If it doesn't help, please check what's the `version` field value in 
- `hello-create-react/node_modules/jest-cli/package.json`
- `hello-create-react/node_modules/jest-util/package.json`

**Comment by ArtemGovorov** [comment]
Looking into your other issue, I have [spotted this](https://github.com/pauleveritt/hello-create-react/blob/master/package.json#L7), which as I suspected is the reason for the issue. You don't need to install `jest-cli`, because `create-react-app` installs it with `react-scripts`.
